clients/admin-ui/cypress/e2e/privacy-experiences.cy.ts

```diff
@@ -556,6 +556,9 @@ describe("Privacy experiences", () => {
         cy.intercept("GET", "/api/v1/experience-config/*", {
           fixture: "privacy-experiences/tcf-experience.json",
         }).as("getTCFExperience");
+        cy.intercept("GET", "/api/v1/privacy-experience/gvl/translations*", {
+          body: {},
+        }).as("getLanguage");
```

Comment:
This was basically the only one that was causing a test failure after the default stubs were added.